src/mochaVErrorReporter.ts

```diff
@@ -0,0 +1,47 @@
+import { Runner, utils as MochaUtils, reporters as MochaReporters } from "mocha";
+import { VError } from "verror";
+
+export class MochaVErrorReporter extends MochaReporters.Base
+{
+    private static _DEFAULT_INDENT: string = "    ";
+    private static _REMOVE_REPEATED_MESSAGE_REGEX: RegExp = /caused by.+\n/;
+    private static _REPLACE_WITH_INDENT_REGEX: RegExp = /\n/g;
+    private static _REMOVE_AFTER_FIRST_LINE_REGEX: RegExp = /\n.+/g;
+    private static stackFilter = MochaUtils.stackTraceFilter();
```

Comment:
You should make this `private static readonly _MOCHA_STACK_FILTER`

Reply:
Fixed